### PR TITLE
ROC-3147 Add bold to rows

### PR DESCRIPTION
**Pull request by robertparkinson**
https://tools.hmcts.net/jira/browse/ROC-3147

Add bold class as an option for row labels for check and send page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/cmc-common-frontend",
-  "version": "1.40.0",
+  "version": "1.41.0",
```
Comment (DamianDunajski) [inline package.json:3]: Why you made that change? Is that needed for that PR? If so update yarn.lock.

**Review by DamianDunajski — COMMENTED** [review]
@robertparkinson would you give us some background for the change? Why is that only for party details component?

Can I ask you to enclose the screenshot of the changed UI as well?

File: package.json (hunk 1 above)
Comment (robertparkinson) [reply package.json:3]: screen shot 2018-03-16 at 08 20 49

it is to match the prototype and it came up in conversation with marc and tarik 
https://hmcts-cmc-prototype.herokuapp.com/claimant/full/c-4/task-list/check-and-send?check=2

**Review by DamianDunajski — DISMISSED** [review]
No update to yarn.lock

**Review by DamianDunajski — APPROVED** [review]

**Pull request merged**